test/loopback.test.js

```diff
@@ -832,7 +832,36 @@ describe('loopback', function() {
       expect(publicMethods).to.be.empty();
     });
 
-    it('hides methods for related models using globs', function() {
+    it('hides methods for related models using globs (model configured first)', function() {
+      var TestModel = app.registry.createModel(uniqueModelName);
+      var RelatedModel = app.registry.createModel(uniqueModelName);
```

Comment:
Both `TestModel` and `RelatedModel` are using the same model name, that's looks suspicious to me. Since we are using local per-app registry now, there is no need to use unique model names anymore.  You can simply do the following:

```js
const TestModel = app.registry.createModel('TestModel');
const RelatedModel = app.registry.createModel('RelatedModel');
// ...
```

(Also note that we are using ES6 syntax now, see http://loopback.io/doc/en/contrib/style-guide.html#variable-declarations)

Reply:
Ah, I see, wasn't aware this was the case. Would like like me to refactor the whole PR or just these is fine?

Reply:
I think fixing it just here (as you have already done) is fine 👍